documentation/source/install_cocotb_windows.rst

```diff
@@ -0,0 +1,18 @@
+Installation of cocotb
+======================
+
+.. versionadded:: 1.2
+
+The **latest release** of cocotb can be installed by running
+
+.. code-block:: bash
+
+    pip install --global-option build_ext --global-option --compiler=mingw32 cocotb
```

Comment:
I don't think that the `--global-option`s are passed to `setup.py` unless you include `--no-use-pep517`.

Reply:
@themperek Do you agree? I don't have Windows...

Reply:
It works here: https://github.com/cocotb/cocotb/blob/master/tox.ini#L25 used on Travis  (does not work/wants use VS) in other case. So it seems it does the job.

Reply:
Ok, leaving this out here; the wiki has it.